mail/mlmmj/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           codeberg 1.0
+PortGroup           legacysupport 1.1
+
+codeberg.setup      mlmmj mlmmj RELEASE_1_4_6
+
+name                mlmmj
+version             1.4.6
+categories          mail
+license             MIT
+maintainers         @inucat \
+                    proton.me:inucat
+description         A simple and slim mailing list manager
+long_description    Mlmmj is a simple and slim mailing list manager (MLM) \
+                    inspired by ezmlm. It works with many different Mail \
+                    Transport Agents (MTAs) and is simple for a system \
+                    adminstrator to install, configure and integrate with \
+                    other software. As it uses very few resources, and \
+                    requires no daemons, it is ideal for installation on \
+                    systems where resources are limited, such as Virtual \
+                    Private Servers (VPSes).
+
+checksums           rmd160  1f462719550aa4cb2410718d30979ebb99c2bb4f \
+                    sha256  5cd877703c80ab07f9ae845b89a34c032c31d290f4a311f6c8b9fe52ca508822 \
+                    size    281169
+patchfiles          include-xlocale_h.diff \
+                    remove-disabled-test.diff
+use_autoreconf      yes
+configure.args       --disable-tests
+configure.env-append LIBS=-liconv
+compiler.blacklist-append *gcc-4.0 *gcc-4.2
```

Comment:
If you need to exclude specific compilers due to some compiler bug or deficiency it's a good idea to add a comment on the line above explaining why, e.g. by showing what error message you got.

If you want to exclude compilers because the code needs a newer C or C++ standard than the compiler supports, instead set `compiler.c_standard` or `compiler.cxx_standard` and let MacPorts pick a suitable compiler.

Reply:
That was me who got the error: https://github.com/macports/macports-ports/pull/24621#issuecomment-2183812449
(I have no idea if the port requires C11 though.)

Reply:
Sorry but I'm not sure what was wrong with these gcc because I didn't use them.
(Using default, maybe it's clang)

Reply:
> Sorry but I'm not sure what was wrong with these gcc because I didn't use them. (Using default, maybe it's clang)

I quote the error in that comment:
```
prepstdreply.c:113: error: redefinition of ‘struct __tllist___COUNTER__’
prepstdreply.c:115: error: redefinition of ‘struct __tllist___COUNTER__’
```
So you could just add these lines above blacklist for old GCC.

Reply:
Added that comment to the Portfile